primes.rb

```diff
@@ -1,6 +1,19 @@
+######CANT LAUNCH SERVER WITHOUT THIS CODE###########
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
-end
\ No newline at end of file
+  def self.sum_to(limit)
+  # TODO - add your prime number solution here...
+    require 'prime'
```

Comment:
Maybe have a go without the core libraries... show us what you can do with Ruby!